src/main/java/jenkins/plugins/clangscanbuild/actions/ClangScanBuildAction.java

```diff
@@ -77,10 +77,10 @@ public ClangScanBuildBugSummary loadBugSummary(){
 		        return null;
 		    }
 	    }catch( java.lang.InterruptedException ie ){
-			System.err.println( ie );
+	    	LOGGER.log(FINEST, "", ie);
 			return null;
 		}catch( IOException ioe ){
-			System.err.println( ioe );
+			LOGGER.log(FINEST, "", ioe);
```

Comment:
I must say that, even when I would set the log level to INFO, I would like to be informed the reason why the summaryXML has not been loaded.  Unlike the `InterruptedException` (an user intended action), this case is a real error, so I would either keep the `System.err` or make it a `LOGGER.log(ERROR, "error reading bug summary XML", ioe);`